replica/table.cc

```diff
@@ -851,7 +851,7 @@ future<> storage_group::split(sstables::compaction_type_options::split opt) {
     }
 
     co_await _main_cg->flush();
-    co_await _main_cg->get_compaction_manager().perform_split_compaction(_main_cg->as_table_state(), std::move(opt));
+    co_await _main_cg->get_compaction_manager().perform_split_compaction(_main_cg->as_table_state(), std::move(opt), tasks::task_info{});
```

Comment:
Why did we pass `std::nullopt` in the first place? Was it a mistake, or a conscious decision?

Reply:
It was a conscious decision. When `compaction_manager::perform_compaction` was added, some compaction types were already covered with tasks and some weren't. I didn't wanted to have compaction executors tasks hanging in task manager without parents.